src/reminder.js

```diff
@@ -0,0 +1,27 @@
+const ms = require('ms');
+
+function handleReminder(message) {
+    if (message.content.startsWith('!remindme')) {
+        const parts = message.content.split(' ');
+
+        if (!parts[1] || isNaN(ms(parta[1]))) {
+            message.channel.send('Not a valid time format...idiot. Remember, the syntax is !remindme (time) (reminder)');
```

Comment:
for (time) are we expecting 0(d):00(hr):00(m):00(s)? Are seconds supported / always expected? Is this restricted to countdown timer only?